hail/www/template.xslt

```diff
@@ -82,36 +82,33 @@
                                 </li>
                             </ul>
                         </div>
+                        <script>
+                            <xsl:text disable-output-escaping="yes" >
+                                <![CDATA[
+                                    (function(){
+                                        var cpage = location.pathname;
+                                        var menuItems = document.querySelectorAll('#hail-menu a');
+
+                                        for (var i = 0; i < menuItems.length; i++) {
+                                            if (menuItems[i].pathname === cpage && menuItems[i].host == location.host) {
+                                                menuItems[i].className = "active";
+                                                return;
+                                            }
+                                        }
+
+                                        if (cpage === "/" || cpage === "/index.html") {
```

Comment:
menu should be matched first to catch the more specific link, since our navbar logo refers to hail.is/ rather than hail.is/docs/0.2.

I'll follow up with a fix to make sure that docs is highlighted for all doc pages.